src/tests/integration/test-data/show/AssertTest.testFail_expect_revert().expected

```diff
@@ -47,64 +47,78 @@
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
-│  (18 steps)
+│  (16 steps)
 ├─ 12
+│   k: #loadProgram b"`\x80`@R4\x80\x15a\x00\x10W`\x00\x80\xfd[P`\x046\x10a\x01BW`\x005 ...
+│   pc: 892
+│   callDepth: 1
+│   statusCode: STATUSCODE:StatusCode
+│
+│  (1 step)
+├─ 13
+│   k: #initVM ~> #precompiled? ( 728815563385977040452943777879061427756277306518 , SH ...
+│   pc: 892
+│   callDepth: 1
+│   statusCode: STATUSCODE:StatusCode
+│
+│  (1 step)
+├─ 14
```

Comment:
Ok, so here we explicitly have the call to `#loadProgram`, triggering the computation of the jumpdests, 18 steps get broken into 16 + 1 + 1.

Reply:
But more importantly, what makes this test so special that it is the only one in which we see the `#loadProgram`?

Reply:
The test is
```Soliditiy
    function testFail_expect_revert() public {
        vm.expectRevert();
        this.call_assert_false();
    }
```
I think that the `this.call_assert_false();` forces the invocation of `call_assert_false` to be an external call instead of an internal call.
The external call leads to `#loadProgram`.

Reply:
Yes, this is correct. That, oddly, means that we have no extrenall calls in the other tests except the CSE ones.